### ISSUE TITLE
Quest: Blueleaf Tubers

### ISSUE THREAD

**Issue by Artanis5** [dalaranwow/dalaran-wow#4685]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Using Crate with Holes in Razorfen Kraul only earns message Nearby tubers required, i have tried it in the normal progress area and the trench below. After some time it stops giving error and does nothing and if you use the command stick (without the gophers around) the error message reappears.

**Expected behaviour**: Tell us what should happen instead

Using the Crate should release gophers and a blue leaf tuber object should appear that is also lootable.

**Steps to reproduce the problem**:

1. Take quest Blueleaf Tubers from Mebok Myzzyrix in Ratchet
2. Head to the Razorfen Kraul
3. Click on Crate with Holes in the trench area

**Include proofs for this behaviour**
http://i.imgur.com/ROE3kBM.jpg
http://i.imgur.com/gFDq0Zt.jpg
http://i.imgur.com/2CgaYP5.jpg
http://i.imgur.com/PRitb7H.jpg

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Im sorry i got no idea how to do that :/.
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello, thank you for the report @Artanis5 .
Issue confirmed, spell http://www.wowhead.com/spell=6918/summon-snufflenose casted by item http://www.wowhead.com/item=5880/crate-with-holes can't be casted and always/anywhere (inside Razorfen Kraul) player gets error "Requires Nearby Tubers" .
Note: Quest works on TC.

Also i would like to add that npc http://www.wowhead.com/npc=4781/snufflenose-gopher summoned by spell http://www.wowhead.com/spell=6918/summon-snufflenose needs to be `set reach on passive` on summon. Currently he is aggressive and if player is high level (creature scales with the owner) he will engage creatures from very far away.

Regarding:

> Im sorry i got no idea how to do that :/.

You can just read:

> You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

find the quest/item/spell/etc and copy the link and paste it here (as i did above), this helps developers a lot!

**Comment by TeridanDW** [comment]
Actually this issue was already fixed here #4381
